antora/supplemental-ui/partials/head-scripts.hbs

```diff
@@ -0,0 +1,3 @@
+<!-- Adobe Analytics for Red Hat - DPAL (DTM Property Auto-Loader) - part 1/2 -->
+<script id="dpal" src="https://www.redhat.com/dtm.js" type="text/javascript"></script>
```

Comment:
Similar to GoogleAnalytics: not sure if you use this; if not, let's rather remove it.